components/delete_channel_modal/delete_channel_modal.test.tsx

```diff
@@ -37,7 +45,7 @@ describe('components/delete_channel_modal', () => {
         channel,
         currentTeamDetails,
         actions: {
-            deleteChannel: emptyFunction,
+            deleteChannel
         },
         onHide: emptyFunction,
```

Comment:
I've noticed some tests using `jest.fn()` instead of `emptyFunction`. Is there any reason not to just use `jest.fn()` which, to my understanding, basically returns an empty mocked function?

Reply:
`jest.fn()` allows our testing library to detect whether a function has been called or not. It's not always necessary, but it's the preferred way of mocking out functions.

Reply:
That makes sense. I've replaced the empty function with `jest.fn()`.